src/components/memorygame/MemoryGameCards/index.jsx

```diff
@@ -1,170 +1,43 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
+import { Col } from "antd";
+import FlippableCard from "../MemoryGameFlippableCard";
+import SuccessCard from "../MemoryGameSuccessCard";
 import "./index.css";
-import ReactCardFlip from "react-card-flip";
-import { Col, Card, Modal, Row } from "antd";
-import { useTranslation } from "react-i18next";
-import questionMark from "../../../images/memoryGameFrontImage.png";
-import memoryGameSuccessSymbol from "../../../images/memoryGameSuccessSymbol.svg";
-import cardInitialStateGenerator from "../cardInitialStateGenerator";
 
-const NUM_REQUIRED_MATCHES = 2;
-// game example: https://www.brightfocus.org/alzheimers/memory-games/match-pictures
-// instead of squares this game uses cards.
-// Card flips on click to show what is behind.
-// matchedCards is an array states that holds successfully matched card states
-// setIsCompleted is a function that is being used to set isCompleted as,
-// boolean state when all possible matches made.
-const MemoryGameCards = ({ setMatchedCards, matchedCards, setIsCompleted }) => {
-  const [t] = useTranslation();
-
-  // cardsDataArr example [{description:"...",link:"...",phrase:".."},...]
-  const cardsDataArr = [
-    ...t("memoryGame.cards", {
-      returnObjects: true,
-    }),
-  ];
-
-  // cardStates is an array of states with multiple properties
-  // state example :
-  // {cardKey:"",img:{src:"",imgKey:""},
-  // isFlipped:false,isMatched:false,description:"",link:"",phrase:""}
-  const [cardStates, setCardStates] = useState(
-    cardInitialStateGenerator(cardsDataArr, NUM_REQUIRED_MATCHES)
-  );
-  // selectedCards is an array of clicked card states(objects).
-  // selectedCards array hold only "NUM_REQUIRED_MATCHES" amount of card states
-  const [selectedCards, setSelectedCards] = useState([]);
-
-  // handle click flips card
-  // if the card is not in selectedCards, adds it to selectedCards
-  const handleClick = (clickedCardState) => {
-    const copyOfCardStates = [...cardStates];
-    // current index is needed to keep the order of cards displayed
-    // because goal is to make user remember the spots of previous cards
-    const currentIndex = cardStates.findIndex(
-      (state) => state.cardKey === clickedCardState.cardKey
-    );
-    const targetCardState = copyOfCardStates[currentIndex];
-    targetCardState.isFlipped = true;
-    setCardStates(() => [...copyOfCardStates]);
-    // prevents user to use same card multiple times
-    const isCardNotInSelectedCards = selectedCards.every(
```

Comment:
louis doesnt like to see negativity in variable name.

Reply:
Red ones are deleted